### PR TITLE
[CodeStyle][Xdoctest][329] Fix example code(`paddle.callbacks.ModelCheckpoint (timeout or error)`)

### PR DESCRIPTION
**Pull request by pxzbest**
### PR Category

User Experience

### PR Types

Docs

### Description

原样例代码使用全部的mnist数据集进行训练，运行时间超过十秒；增加的运行时间

### CODE REVIEW

File: python/paddle/hapi/callbacks.py [hunk 1]
```diff
@@ -647,33 +647,40 @@ class ModelCheckpoint(Callback):
...
-
+        .. code-block:: pycon
+        
             >>> import paddle
             >>> import paddle.vision.transforms as T
             >>> from paddle.vision.datasets import MNIST
             >>> from paddle.static import InputSpec
+            >>> # doctest: +TIMEOUT(80)
 
+            >>> paddle.seed(2023)
```
Comment (SigureMo) [inline python/paddle/hapi/callbacks.py]: 这个加 seed 的必要是？

File: python/paddle/hapi/callbacks.py [hunk 2]
```diff
@@ -647,33 +647,40 @@ class ModelCheckpoint(Callback):
...
     Examples:
-        .. code-block:: python
-
+        .. code-block:: pycon
+        
             >>> import paddle
             >>> import paddle.vision.transforms as T
             >>> from paddle.vision.datasets import MNIST
             >>> from paddle.static import InputSpec
+            >>> # doctest: +TIMEOUT(80)
```
Comment (SigureMo) [inline python/paddle/hapi/callbacks.py]: 放最上面

File: python/paddle/hapi/callbacks.py [hunk 3]
```diff
@@ -647,33 +647,40 @@ class ModelCheckpoint(Callback):
...
+            ...     optimizer=optim,
+            ...     loss=paddle.nn.CrossEntropyLoss(),
+            ...     metrics=paddle.metric.Accuracy()
+            ... )
 
             >>> callback = paddle.callbacks.ModelCheckpoint(save_dir='./temp')
-            >>> model.fit(train_dataset, batch_size=64, callbacks=callback)
+            >>> model.fit(train_dataset, batch_size=2, epochs=1, callbacks=callback, verbose=0)
+            >>> print('Fit finished')
+            Fit finished
```
Comment (SigureMo) [inline python/paddle/hapi/callbacks.py]: 删掉

**Review by SigureMo — COMMENTED** [review]
格式化代码，通过 codestyle 流水线

**Comment by SigureMo** [comment]
> [CodeStyle][Xdoctest][1] Fix example code(`paddle.callbacks.ModelCheckpoint (timeout or error)`)

你这是任务 1 吗？

**Comment by PangXianzheng** [comment]
> > [CodeStyle][Xdoctest][1] Fix example code(`paddle.callbacks.ModelCheckpoint (timeout or error)`)
> 
> 你这是任务 1 吗？

 我这个是任务329

**Comment by SigureMo** [comment]
> `[1]`

那为什么写 `[1]`？

**Comment by PangXianzheng** [comment]
> > `[1]`
> 
> 那为什么写 `[1]`？

不好意思，是我的失误，我修改一下

File: python/paddle/hapi/callbacks.py [hunk 4]
```diff
@@ -647,33 +647,30 @@ class ModelCheckpoint(Callback):
...
 
+            >>> paddle.seed(2023)
             >>> inputs = [InputSpec([-1, 1, 28, 28], 'float32', 'image')]
             >>> labels = [InputSpec([None, 1], 'int64', 'label')]
 
-            >>> transform = T.Compose([
-            ...     T.Transpose(),
-            ...     T.Normalize([127.5], [127.5])
-            ... ])
+            >>> transform = T.Compose([T.Transpose(), T.Normalize([127.5], [127.5])])
```
Comment (SigureMo) [inline python/paddle/hapi/callbacks.py]:
```suggestion
            >>> transform = T.Compose([T.Transpose(), T.Normalize([127.5], [127.5])],)
```

太长了看不清楚，添加 magic trailing comma 后重新格式化

File: python/paddle/hapi/callbacks.py [hunk 5]
```diff
@@ -647,33 +647,30 @@ class ModelCheckpoint(Callback):
...
             >>> lenet = paddle.vision.models.LeNet()
-            >>> model = paddle.Model(lenet,
-            ...     inputs, labels)
+            >>> model = paddle.Model(lenet, inputs, labels)
 
             >>> optim = paddle.optimizer.Adam(0.001, parameters=lenet.parameters())
-            >>> model.prepare(optimizer=optim,
-            ...             loss=paddle.nn.CrossEntropyLoss(),
-            ...             metrics=paddle.metric.Accuracy())
+            >>> model.prepare(optimizer=optim, loss=paddle.nn.CrossEntropyLoss(), metrics=paddle.metric.Accuracy())
```
Comment (SigureMo) [inline python/paddle/hapi/callbacks.py]: 同上

File: python/paddle/hapi/callbacks.py [hunk 6]
```diff
@@ -647,33 +647,30 @@ class ModelCheckpoint(Callback):
...
 
             >>> callback = paddle.callbacks.ModelCheckpoint(save_dir='./temp')
-            >>> model.fit(train_dataset, batch_size=64, callbacks=callback)
+            >>> model.fit(train_dataset, batch_size=2, epochs=1, callbacks=callback, verbose=0)
```
Comment (SigureMo) [inline python/paddle/hapi/callbacks.py]: 同上

File: python/paddle/hapi/callbacks.py (hunk 1 above)
Comment (SigureMo) [reply python/paddle/hapi/callbacks.py]: 这里？

File: python/paddle/hapi/callbacks.py [hunk 7]
```diff
@@ -657,23 +658,30 @@ class ModelCheckpoint(Callback):
             >>> inputs = [InputSpec([-1, 1, 28, 28], 'float32', 'image')]
             >>> labels = [InputSpec([None, 1], 'int64', 'label')]
 
-            >>> transform = T.Compose([
-            ...     T.Transpose(),
-            ...     T.Normalize([127.5], [127.5])
-            ... ])
+            >>> transform = T.Compose(
+            ...     [T.Transpose(), T.Normalize([127.5], [127.5])],
```
Comment (SigureMo) [inline python/paddle/hapi/callbacks.py]:
```suggestion
            ...     [T.Transpose(), T.Normalize([127.5], [127.5]),],
```

应该是这里加，上一个搞错了

**Review by SigureMo — APPROVED** [review]

**Comment by SigureMo** [comment]
@ooooo-create 要不豁免下？docs preview 是 PaddlePaddle/PaConvert#807 搞挂的

**Comment by luotao1** [comment]
@pxzbest 有兴趣可以参加如下活动：
- https://github.com/PaddlePaddle/Paddle/issues/77510
